tests/recipient_validation/test_phone_number.py

```diff
@@ -383,19 +385,19 @@ def test_format_uk_and_international_phone_numbers(phone_number, expected_format
         ("foo", "foo"),
         ("TeSt@ExAmPl3.com", "test@exampl3.com"),
         ("+4407900 900 123", "447900900123"),
-        ("+1 800 555 5555", "18005555555"),
+        ("+1 928-282-4541", "19282824541"),
     ],
 )
 def test_format_recipient(recipient, expected_formatted):
     assert format_recipient(recipient) == expected_formatted
 
 
-def test_try_format_recipient_doesnt_throw():
-    assert try_validate_and_format_phone_number("ALPHANUM3R1C") == "ALPHANUM3R1C"
+# def test_try_format_recipient_doesnt_throw():
+#     assert try_validate_and_format_phone_number("ALPHANUM3R1C") == "ALPHANUM3R1C"
 
 
-def test_format_phone_number_human_readable_doenst_throw():
-    assert format_phone_number_human_readable("ALPHANUM3R1C") == "ALPHANUM3R1C"
+# def test_format_phone_number_human_readable_doenst_throw():
+#     assert format_phone_number_human_readable("ALPHANUM3R1C") == "ALPHANUM3R1C"
```

Comment:
currently commented out, confirming we can delete as we don't have an equivalent method. Invalid characters would be caught at the instantiation of the object, not during a try_and_validate_and_format_method

Reply:
It looks like we are using this a lot in admin repo: https://github.com/search?q=repo%3Aalphagov%2Fnotifications-admin%20format_phone_number_human_readable&type=code

So maybe we actually we do need it? Or an equivalent?

Reply:
As here https://github.com/alphagov/notifications-utils/pull/1158#discussion_r1819575413